### PR TITLE
Besties stack

### PR DESCRIPTION
**Pull request by orangewolf**

### CODE REVIEW

File: ops/demo-deploy.tmpl.yaml [hunk 1]
```diff
@@ -1,19 +1,19 @@
 replicaCount: 2
 
 imagePullSecrets:
-  - name: github-registry
+  - name: github
```
Comment (aprilrieger) [inline ops/demo-deploy.tmpl.yaml:4]: github-registry please :)

File: ops/demo-deploy.tmpl.yaml [hunk 2]
```diff
@@ -59,27 +59,27 @@ extraEnvVars: &envVars
...
-    value: staging-postgresql
+    value: postgres-cluster-alpha-ha.postgres.svc.cluster.local
   - name: DB_NAME
-    value: hyku
+    value: pals-demo-hyku
   - name: DB_PASSWORD
     value: $DB_PASSWORD
-  - name: DATABASE_URL
-    value: postgresql://postgres:$DB_PASSWORD@pg-postgresql.staging-postgres.svc.cluster.local/staging-palni-palci?pool=5
+  - name: DB_URL
```
Comment (aprilrieger) [inline ops/demo-deploy.tmpl.yaml:67]: Saw your comment on utk-hyku prod deploy, so does this need to be defined here?

Comment (orangewolf) [reply ops/demo-deploy.tmpl.yaml:67]: it does not

File: ops/production-deploy.tmpl.yaml [hunk 3]
```diff
@@ -0,0 +1,329 @@
+replicaCount: 2
+
+imagePullSecrets:
+  - name: github
```
Comment (aprilrieger) [inline ops/production-deploy.tmpl.yaml:4]: github-registry

File: ops/demo-deploy.tmpl.yaml (hunk 1 above)
Comment (orangewolf) [reply ops/demo-deploy.tmpl.yaml:4]: we agreed to leave this as is

**Pull request merged**